packages/camera/camera_avfoundation/example/lib/main.dart

```diff
@@ -599,18 +591,27 @@ class _CameraExampleHomeState extends State<CameraExampleHome>
             width: 90.0,
             child: RadioListTile<CameraDescription>(
               title: Icon(getCameraLensIcon(cameraDescription.lensDirection)),
-              groupValue: controller?.description,
               value: cameraDescription,
-              onChanged: (controller?.value.isRecordingVideo ?? false)
```

Comment:
This is a behavioral change. Setting `onChanged` to null was disabling the control, and the replacement code isn't ever disabling it.